File: app/src/main/java/com/letianpai/robot/wificonnet/parser/VersionInfo.java
package com.letianpai.robot.wificonnet.parser;

/**
 * @author liujunbin
 */
public class VersionInfo {
    private int code;
    private String msg;
    private VersionData data;

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public VersionData getData() {
        return data;
    }

    public void setData(VersionData data) {
        this.data = data;
    }

    @Override
    public String toString() {
        return "{" +
                "code:" + code +
                ", msg:'" + msg + '\'' +
                ", data:" + data +
                '}';
    }
}

File: app/src/main/java/com/letianpai/robot/wificonnet/parser/VersionInfo.kt
package com.letianpai.robot.wificonnet.parser

/**
 * @author liujunbin
 */
class VersionInfo {
    @JvmField
    var code: Int = 0
    var msg: String? = null
    @JvmField
    var data: VersionData? = null

    override fun toString(): String {
        return "{" +
                "code:" + code +
                ", msg:'" + msg + '\'' +
                ", data:" + data +
                '}'
    }
}
